test/benchmarks/run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
We should use at least `set -e` everywhere by default. Or better, the full set:

```suggestion
#!/usr/bin/env bash
set -euo pipefail
```